lib/Elevate/Leapp.pm

```diff
@@ -51,13 +51,29 @@ sub install ($self) {
         $self->yum->install( 'leapp-upgrade', $leapp_data_pkg );
     }
 
+    return;
+}
+
+sub remove_kernel_devel ($self) {
+
     if ( Cpanel::Pkgr::is_installed('kernel-devel') ) {
         $self->yum->remove('kernel-devel');
     }
 
     return;
 }
 
+sub preupgrade ($self) {
+
+    INFO("Running leapp preupgrade checks");
+
+    $self->cpev->ssystem_hide_output( '/usr/bin/leapp', 'preupgrade' );
```

Comment:
Do we need the `--nowarn` flag for CloudLinux here?  It was required for `upgrade`, but I'm not sure about `preupgrade`

Reply:
I'm not sure about that one.

Reply:
Sounds good.  Thank you for confirming this 👍